gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -1306,6 +1307,83 @@ gum_do_enumerate_exports (GumEnumerateExportsContext * ctx,
   return carry_on;
 }
 
+kern_return_t
+gum_malloc_memory_reader (task_t remote_task,
+    vm_address_t remote_address, vm_size_t size, void ** local_memory) {
+
+  *local_memory = (void *) remote_address;
+
+  return KERN_SUCCESS;
+}
+
+struct _gum_malloc_helper_info {
+  GumFoundMallocRangeFunc func;
+  gpointer user_data;
+  gboolean should_abort;
+};
```

Comment:
Please also move structs to the top of the file together with the other structs (in chronological order).